### PR TITLE
[CREATE ABOUT US COMPONENT]

### PR DESCRIPTION
**Pull request by RomeoPetryshyn**
Created component 'About us' with user-friendly interface, with the general information about our project and about creators of the project. Also added 2 charts about project front-end statistic based on github data(was done using github api). Also created a list of creators of the project with more detailed information about what was done and who did it

### CODE REVIEW

File: src/app/admin/about-us/bars-chart/bars-chart.component.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    const nickNames = data.map(item => {
+      return item.author.login;
+    });
+    return nickNames;
+  }
+
+  private async getUserNames() {
+    const data: any = await this.getUserNickNames();
+    data.forEach(element => {
+      this.aboutUsService.getUserNames(element).subscribe((result: any) => {
```
Comment (AndriyIf) [inline src/app/admin/about-us/bars-chart/bars-chart.component.ts]: it is better to use forkJoin here. 
const someConst$ = data.map(element) => this.aboutUsService.getUserNames(element))
forkJoin(someConst$).subscribe(result => this.barChartLabels = result)
or you could use promise.all
const someConst = data.map(element) => this.aboutUsService.getUserNames(element)).toPromise()
Promise.all(someConst).then(result => this.barChartLabels = result)

File: src/app/admin/students/students-modal-window/students-modal-window.component.ts [hunk 2]
```diff
@@ -137,11 +137,7 @@ export class StudentsModalWindowComponent implements OnInit {
   }
 
   hidePassword() {
-    if (this.hide === true) {
-      return this.hide = false;
-    } else {
-      return this.hide = true;
-    }
+    return this.hide = this.hide === true ? false : true;
```
Comment (AndriyIf) [inline src/app/admin/students/students-modal-window/students-modal-window.component.ts]: this.hide = !this.hide

File: src/app/admin/about-us/bars-chart/bars-chart.component.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    this.getCommitsAmount();
+  }
+
+  private async getContributorsInfo() {
+    return this.aboutUsService.getContributorsData().toPromise();
+  }
+
+  private async getUserNickNames() {
+    const data: any = await this.getContributorsInfo();
+    const nickNames = data.map(item => {
```
Comment (AndriyIf) [inline src/app/admin/about-us/bars-chart/bars-chart.component.ts]: It is better to use contractions 
`const data: any = await this.getContributorsInfo();
return data.map(({author}) => author.login)`
or 
`
return this.getContributorsInfo()
   .then((users) => users.map(({author}) => author.login))
`

File: src/app/admin/students/students-modal-window/students-modal-window.component.ts (hunk 2 above)
Comment (RomeoPetryshyn) [reply src/app/admin/students/students-modal-window/students-modal-window.component.ts]: Fixed

File: src/app/admin/about-us/bars-chart/bars-chart.component.ts (hunk 3 above)
Comment (RomeoPetryshyn) [reply src/app/admin/about-us/bars-chart/bars-chart.component.ts]: Tried both ways - no one is working

File: src/app/admin/about-us/bars-chart/bars-chart.component.ts (hunk 1 above)
Comment (RomeoPetryshyn) [reply src/app/admin/about-us/bars-chart/bars-chart.component.ts]: Fixed using the suggestion with forkJoin

File: src/app/admin/about-us/bars-chart/bars-chart.component.ts (hunk 3 above)
Comment (RomeoPetryshyn) [reply src/app/admin/about-us/bars-chart/bars-chart.component.ts]: Fixed using second suggestion

**Review by yurakhomitsky — APPROVED** [review]

**Review by roman89ua — APPROVED** [review]

**Pull request merged**